src/reducers/fileTree.spec.tsx

```diff
@@ -1012,4 +1017,183 @@ describe(__filename, () => {
       });
     });
   });
+
+  describe('findRelativePathWithDiff', () => {
+    const file1 = 'file1.js';
+    const file2 = 'file2.js';
+    const file3 = 'file3.js';
+    const file4 = 'file4.js';
+
+    // const entries = [
```

Comment:
👻

Reply:
I guess we don't have an `eslint` rule to catch stuff like that. I usually spot it after pushing but I guess I didn't check the full diff this time.

Reply:
This is what code review is good for 😎